src/applications/gqrx/mainwindow.cpp

```diff
@@ -1588,7 +1588,7 @@ void MainWindow::stopIqRecording()
         ui->statusBar->showMessage(tr("I/Q data recoding stopped"), 5000);
 }
 
-void MainWindow::startIqPlayback(const QString& filename, float samprate)
+void MainWindow::startIqPlayback(const QString& filename, float samprate, double center_freq)
```

Comment:
`qint64` is used elsewhere in Gqrx for frequencies, so I think we should use that instead of `double` or `long long` (below).

Reply:
Done.